view/frontend/templates/checkout/success.phtml

```diff
@@ -25,7 +25,28 @@
  * @var \Adyen\Payment\Block\Checkout\Success $block
  */
 ?>
-<?php if ($block->isBoletoPayment()): ?>
+<?php if ($block->renderAction()): ?>
+    <script type="text/javascript">
+        require([
+            'jquery',
+            'adyenCheckout3101'
+        ], function ($, AdyenCheckout) {
+            var action = JSON.parse('<?= $block->getAction(); ?>');
```

Comment:
I don't know if we can escape it or not so to be sure that we don't change the structure of the action I would just add this:

```suggestion
            var action = JSON.parse('<?= /* @noEscape */ $block->getAction(); ?>');
```

Reply:
Would be nice to add some checks on the backend as well, then.

Reply:
We can add this tag. @msilvagarcia not sure what you mean by add checks in the backend. As this comes from the API we just need to pass it to the component. The component will use it there will be no server call afterwards.